### DISCUSSION TITLE
请问最后一版backend是shadowsocks-libev的版本号是?

### DISCUSSION THREAD

**Discussion by sophauer** [shadowsocks/shadowsocks-android#2778] [Q&A]
as tittle,
自从shadow-rust剔除xchacha20-ietf-poly1305又加回去,个人体验起来兼容有点问题
但不想把加密方式改成chacha20-ietf-poly1305.
想换回以前的shadowsocks-libev做backend的版本,
看了一圈release,实在看不出哪一版本是最后一个版本
故求教

**Comment by loyess** [comment]
5.0.6
